libraries/wallet/wallet.cpp

```diff
@@ -2630,8 +2770,10 @@ fc::ecc::private_key wallet_api::derive_private_key(const std::string& prefix_st
             const std::string& owner, const std::string& contact,
             private_contact_type type, fc::optional<std::string> json_metadata, bool broadcast
         ) {
-            FC_ASSERT(!is_locked());
-            FC_ASSERT(type != golos::plugins::private_message::undefined || !json_metadata);
+            WALLET_CHECK_UNLOCKED();
+            GOLOS_CHECK_PARAM(type,
+                GOLOS_CHECK_VALUE(type != golos::plugins::private_message::undefined || !json_metadata,
```

Comment:
It's not related to current changes, but to previous PR.
"`undefined` contact" sounds strange (especially for end user), maybe rename it to `unknown` or simply `normal`

Reply:
Renamed to `unknown`.